application-portal/src/app/pages/courses/courses.page.html

```diff
@@ -5,36 +5,18 @@
 </ion-header> -->
 
 <ion-content class="courses-page">
-  <div class="cool-header"><b>Nice. You have three tech-focused course-internship programs to choose from this summer.</b></div>
+  <div class="cool-header"><b>Nice. You have {{numberOfCourses}} {{field}}-focused course-internship program(s) to choose from this summer.</b></div>
```

Comment:
See if you can detect whether it's 1 or more for programs

Hint: `ng-pluralize`